pulp.spec

```diff
@@ -967,8 +967,8 @@ SELinux policy for Pulp's components
 %pre selinux
 # Record old version so we can limit which restorecon statement are executed later
 test -e %{_localstatedir}/lib/rpm-state/%{name} || mkdir -p %{_localstatedir}/lib/rpm-state/%{name}
-oldversion=$(semodule -l | grep pulp-server)
-echo ${oldversion:12} > %{_localstatedir}/lib/rpm-state/%{name}/old-version
+oldversion=$(rpm -qa pulp-selinux)
+echo ${oldversion:13} > %{_localstatedir}/lib/rpm-state/%{name}/old-version
```

Comment:
Line 971 takes a string that looks like 'pulp-selinux-2.10.0-0.1' and writes just '2.10.0-0.1' to a file called old-verion. This is done before the uninstalling the pulp-selinux package. The content of that file is then used in the post install step as the argument for the relabel.sh script.